src/4.3_to_5.0/_cleanupDescendants.test.ts

```diff
@@ -60,7 +60,7 @@ describe("_cleanupDescendants", () => {
       NON EMPTY Hierarchize(
             Descendants(
               {
-                [Booking].[Desk].[ALL].[AllMember].[LegalEntityA]
+                [Booking].[Desk].[ALL].[AllMember].[LegalEntityA].[BusinessUnitA].[DeskA].[0]
```

Comment:
We were actually checking something invalid with the previous input: 
``` 
Descendants(
              {
                [Booking].[Desk].[ALL].[AllMember].[LegalEntityA]
               },
               1
)
```
should NOT be replaced by just `LegalEntityA`.
Indeed the level at distance 1 from `LegalEntityA` is `BusinessUnit`.

However in the new test input:
``` 
Descendants(
              {
                [Booking].[Desk].[ALL].[AllMember].[LegalEntityA].[BusinessUnitA].[DeskA].[0]
               },
               1
)
```
the Descendants node should be replaced by the `0` BookId, since it doesn't have descendants anyway.